src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -155,10 +155,334 @@ public SyntaxNode MethodDeclaration(IMethodSymbol method, IEnumerable<SyntaxNode
         }
 
         /// <summary>
-        /// Creates a method declaration.
+        /// Creates an implicit conversion declaration.
+        /// </summary>
+        public virtual SyntaxNode ImplicitConversionDeclaration(
+            IEnumerable<SyntaxNode> parameters = null,
+            SyntaxNode returnType = null,
+            Accessibility accessibility = Accessibility.NotApplicable,
+            DeclarationModifiers modifiers = default(DeclarationModifiers),
+            IEnumerable<SyntaxNode> statements = null)
+        {
+            throw new NotImplementedException();
```

Comment:
As I said, it's a good practice to associate this with a capability API, and I encourage you to do that here. I'd say 'never' is too strong, though. There are many places in practice in the framework where NotSupported can occur without a corresponding capability API.